pkg/service/presentation/service.go

```diff
@@ -27,6 +27,8 @@ import (
 	"github.com/tbd54566975/ssi-service/pkg/storage"
 )
 
+const presentationRequestNamespace = "presentation_request"
```

Comment:
have we gone with snake case for other namespaces?

Reply:
It's been a bit all over the place, created https://github.com/TBD54566975/ssi-service/issues/506 to improve.